firestore.rules

```diff
@@ -0,0 +1,23 @@
+service cloud.firestore {
+  match /databases/{database}/documents {
+    match /company/{document=**} {
+    	allow read;
+    }
+    
+    match /company/{userId} {
+      allow update: if request.auth.uid == userId;
+      allow create: if request.auth.uid != null;
+    }
+    
+    match /offer/{document=**} {
+    	function isOwner() {
+        return request.auth.uid == request.resource.data.author_uid;
```

Comment:
Please, fix indentation in this file.

Reply:
all indentation are ok

Reply:
Github shows me something different here:
![image](https://user-images.githubusercontent.com/17861616/54184782-bb0e1400-44a7-11e9-80fd-7fabdf660a1b.png)
But maybe it's github internal problem, nevertheless, this is a minor problem.